docs/src/main/sphinx/sql/deny.rst

```diff
@@ -0,0 +1,51 @@
+====
+DENY
+====
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    DENY ( privilege [, ...] | ( ALL PRIVILEGES ) )
+    ON ( table_name | TABLE table_name | SCHEMA schema_name)
+    TO ( user | USER user | ROLE role )
+
+Description
+-----------
+
+Denies the specified privileges to the specified grantee.
+
+Specifying ``ALL PRIVILEGES`` denies :doc:`delete`, :doc:`insert`, :doc:`update` and :doc:`select` privileges.
+
+Specifying ``ROLE PUBLIC`` denies privileges to the ``PUBLIC`` role and hence to all users.
```

Comment:
Sysadmin is excluded from this, for obvious reasons. (If this also affected sysadmin, the DENY grant would be impossible to revoke.)